src/hotspot/cpu/s390/vmreg_s390.cpp

```diff
@@ -43,6 +43,13 @@ void VMRegImpl::set_regName() {
     regName[i++] = freg->name();
     freg = freg->successor();
   }
+
+  VectorRegister vreg = ::as_VectorRegister(0);
+  for (; i < ConcreteRegisterImpl::max_vr;) {
```

Comment:
NIT: this really looks like a `while` loop.

Reply:
@JohnTortugo Could you elaborate more on this?

Reply:
I think this is what he meant: 
```suggestion
  while(i < ConcreteRegisterImpl::max_vr) {
```